modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -216,6 +216,89 @@ input ConstraintSetInput {
   elevationRange: ElevationRangeInput
 }
 
+
+
+
+
+
+
+
+scalar ChronicleId
+scalar TransactionId
+
+interface ChronicleEntry {
+  id: ChronicleId!
+  transactionId: TransactionId
+  user: User
+  timestamp: Timestamp!
+}
+
+type ConditionsEntry { # implements ChronicleEntry
+  id: ChronicleId!
+  transactionId: TransactionId
+  user: User
+  timestamp: Timestamp!
+  measurement: ConditionsMeasurement
+  intuition: ConditionsIntuition
+}
+
+input ConditionsEntryInput {
+  measurement: ConditionsMeasurementInput
+  intuition: ConditionsIntuitionInput
+}
+
+"Non-negative floating-point value."
+scalar Extinction
+
+type ConditionsMeasurement {
+  source:     ConditionsMeasurementSource!
+  seeing:     Angle
+  extinction: Extinction
+  wavelength: Wavelength
+  azimuth:    Angle
+  elevation:  Angle
+}
+
+input ConditionsMeasurementInput {
+  source:     ConditionsMeasurementSource!
+  seeing:     AngleInput
+  extinction: Extinction
+  wavelength: WavelengthInput
+  azimuth:    AngleInput
+  elevation:  AngleInput
+}
+
+type ConditionsIntuition {
+  expectation: ConditionsExpectation
+  seeingTrend: SeeingTrend
```

Comment:
Is `SeeingTrend` defined here?  I don't see it.

Reply:
The three enumerated types here seemed pretty wishy-washy so I defined them in tables.

Reply:
It doesn't need some kind of definition in the schema though?

Reply:
No, the enum type is added to the schema in `OdbMapping` when it smooshes the schema source with `enums.schema`

```scala
          val schema: Schema =
            unsafeLoadSchema("OdbSchema.graphql") |+| enums.schema
```

Reply:
In a just world you couldn't construct a schema with missing types, but Grackle doesn't mind.